SFUsat/sfu_logging_task.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * sfu_logging_queue.c
```

Comment:
Again for consistency, the files should be renamed to `sfu_task_logging.c` etc, and comments such as these should be updated appropriately.